### ISSUE TITLE
"Endorsing node" selector sequence or options may be confusing (?)

### ISSUE THREAD

**Issue by gbif-portal** [gbif/portal-feedback#633]
**&quot;Endorsing node&quot; selector sequence or options may be confusing (?)**

We have a single case where a new publisher took the "Request endorsement for OBIS data" to be the default. This being an isolated case so far, it probably does not require action yet, but I'd like to watch whether this happens in other cases as well.

-----
fbitem-ab3ccb02d409d7c1b6652ceee60e3937ce068b3e
User provided contact info: ahahn-gbif
System: Firefox 56.0.0 / Windows 10 0.0.0
Referer: https://www.gbif.org/become-a-publisher
Window size: width 1603 - height 986
[API log](http://elk.gbif.org:5601/app/kibana#/discover/UAT-Varnish-403s?_g=(refreshInterval:(display:Off,pause:!f,value:0),time:(from:'2017-11-03T10:11:50.662Z',mode:absolute,to:'2017-11-03T10:17:50.662Z'))&_a=(columns:!(request,response,clientip),filters:!(),index:'prod-varnish-*',interval:auto,query:(query_string:(analyze_wildcard:!t,query:'response:%3E499%20AND%20(request:%22%2F%2Fapi.gbif.org%22)')),sort:!('@timestamp',desc))&indexPattern=uat-varnish-*&type=histogram)
[Site log](http://elk.gbif.org:5601/app/kibana#/discover/UAT-Varnish-403s?_g=(refreshInterval:(display:Off,pause:!f,value:0),time:(from:'2017-11-03T10:11:50.662Z',mode:absolute,to:'2017-11-03T10:17:50.662Z'))&_a=(columns:!(request,response,clientip),filters:!(),index:'prod-varnish-*',interval:auto,query:(query_string:(analyze_wildcard:!t,query:'response:%3E399%20AND%20(request:%22%2F%2Fdemo.gbif.org%22)')),sort:!('@timestamp',desc))&indexPattern=uat-varnish-*&type=histogram)

**Comment by MortenHofft** [comment]
Good idea to keep an eye on it, but given that this is the view I would think it was pretty clear what you had selected (this is the default shown btw). If you see this happen enough to act on it, then I might need some assistance in how to make it more obvious.

![screen shot 2017-11-03 at 13 43 57](https://user-images.githubusercontent.com/6759267/32374269-3177bd16-c09d-11e7-9fc6-a27d2f2e748c.png)

**Comment by ahahn-gbif** [comment]
We got similar case on helpdesk where the publisher writes: "we need to register an organization under the auspices of the Ministry of the Environment, but in the link (https://www.gbif.org/become-a-publisher) it does not appear under Endorsing node section". 

Given that there is no endorsing node section to select a country from, only the "or select another associated organisation" dropdown, and that the country is marked as the default (provided the information in the sections above does name it), there appears to be confusion between the country as an associate participant and the ministery as the organization hosting the node. Still watching this issue, but one option would be to be clearer on the terminology of "Participant Node" as "Country Participant (Node)" and "Other Associated Participant Node", rather than "associated organization".

![ecuador](https://user-images.githubusercontent.com/13435656/32884104-8a9bded8-cab9-11e7-96b0-b00d4ec5ec49.png)

**Comment by ahahn-gbif** [comment]
Another case where the endorsement request was sent to OBIS, though there is no marine connection: https://www.gbif.org/publisher/ee40cf11-c430-4530-80ac-71dad1bd5166. Got notified by Dag, who confirms alternative Norwegian endorsement instead.

**Comment by MortenHofft** [comment]
Perhaps it is the wording? It tells you to do an action, but in reality we have preselected what we believe the user should do so the user shouldn't do anything.

**Comment by ahahn-gbif** [comment]
Yet another one where the request came to us via OBIS, this time Botswana (https://www.gbif.org/publisher/02b89818-33ab-4bf7-984d-f88a8e0b0b0f).

Consider indeed changing the wording:
Under "Endorsing node", 
- keep the first sentence ("To support... with a GBIF node."), but change the second to "Please check the suggestion below, and correct it if needed:"
- change the text behind the OBIS radio button to "Marine data publishers: request endorsement for OBIS related data", with either a link behind "OBIS" pointing at http://www.iobis.org/, or the full name in brackets (Ocean Biogeographic Information System)

**Comment by ahahn-gbif** [comment]
Over the last months, this has no longer been an issue. Closing.

**Comment by ahahn-gbif** [comment]
Reopening:
Following a recent mail exchange on helpdesk (9.4.19), there is still confusion where US publishers do not follow the suggestion, but select iDigBio from the dropdown in cases where the endorsement should be done by the US country node (USGS), not by an associate participant. Contributing to the confusion is the fact that some Associate Participants are also registered as publishers / hosts, so that the impression arises that some hosts are allowed to endorse publishers, while others are not.

Different approaches have been loosely discussed:
- remove iDigBio from the dropdown list at https://www.gbif.org/become-a-publisher
-- should we really handle iDigBio any different from other associate participants? Do we need different classes of associate participants, as far as the right to endorse is concerned? Should all but a few selected (e.g. already endorsing) associate participants be removed, and only those added again that request it? Who needs to discuss / decide this?
- make the participant role more prominent, e.g. in the suggestion, add "U.S. Geological Survey: US Country Participant Node", or in the dropdown "iDigBio - Associate Participant node"
- move the dropdown list one step "deeper" to make it a conscious refusal of the "Suggested" entry, e.g. "If endorsement through the country node suggested above is not the right option, please check this list of associated participants for multinational or thematic networks:"

**Comment by albenson-usgs** [comment]
Yes this is occasionally still an issue in the US. I think iDigBio would be ok with removing them from the drop down list as David Jennings just has to forward to me and the help desk any time someone selects iDigBio. I don't think there is a ever a use case where iDigBio would be the endorsing node. From what I understand Specify is also an option? I don't know what it looks like from the data publisher end of things when they try to register as I've never done it.
